nautobot/dcim/templates/dcim/softwareimage_retrieve.html

```diff
@@ -0,0 +1,174 @@
+            <div class="row">
+                <div class="col-md-12">
+                    <div class="panel panel-default">
+                        <div class="panel-heading">
+                            <strong>Software Image</strong>
+                        </div>
+                        <div class="table-responsive">
+                            <table class="table table-hover panel-body attr-table">
+                                <tr>
+                                    <td>Status</td>
+                                    <td>
+                                        {{ object.status|hyperlinked_object_with_color }}
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td>Software Version</td>
+                                    <td>{{ object.software_version|hyperlinked_object }}</td>
+                                </tr>
+                                <tr>
+                                    <td>Image File Name</td>
+                                    <td>
+                                        {% if object.image_file_name %}
+                                        <span class="hover_copy">
+                                            <span id="copy_image_file_name">{{ object.image_file_name }}</span>
+                                            <button type="button" class="btn btn-inline btn-default hover_copy_button" data-clipboard-target="#copy_image_file_name">
+                                                <span class="mdi mdi-content-copy"></span>
+                                            </button>
+                                        </span>
+                                        {% else %}
+                                        <span class="text-muted">&mdash;</span>
+                                        {% endif %}
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td>Image File Checksum</td>
+                                    <td>
+                                        {% if object.image_file_checksum %}
+                                        <span class="hover_copy">
+                                            <span id="copy_image_file_checksum">{{ object.image_file_checksum }}</span>
+                                            <button type="button" class="btn btn-inline btn-default hover_copy_button" data-clipboard-target="#copy_image_file_checksum">
+                                                <span class="mdi mdi-content-copy"></span>
+                                            </button>
+                                        </span>
+                                        {% else %}
+                                        <span class="text-muted">&mdash;</span>
+                                        {% endif %}
```

Comment:
Would be really good to add a templatetag that renders this repeated pattern for copyable fields.

Reply:
I was going to write one but can't remember why I didn't..